core/tools.js

```diff
@@ -945,6 +945,8 @@
 		/**
 		 * Converts the specified CSS length value to the calculated pixel length inside this page.
 		 *
+		 * Since 4.10.1 it returns also negative values.
+		 *
 		 * **Note:** Percentage-based value is left intact.
 		 *
 		 * @method
```

Comment:
I know that it's not directly connected with the issue, but we could add docs about return type  – `Number/String` (string is returned for percentage values).